### PR TITLE
Various changes from redesign checklist

### PR DESCRIPTION
**Pull request by vincentbavitz**
- [X] Fix adding profile picture
- [X] Generalise font-size to constants in SASS
- [X] Remove inverted QR codes
- [X] Put version number in settings view → about 
- [X] Need to make these buttons SessionButtonType.Default to match all the other modals. Also need to capitalise the message.
- [X] User details dialog on clicking top left avatar, show QR etc.
- [X] Put “link new device” button behind password lock
- [X] Allow building of AppImages for release

### CODE REVIEW

File: js/background.js [hunk 1]
```diff
@@ -972,6 +973,9 @@
       return toastID;
     };
 
+    // Get memberlist. This function is not accurate >>
+    // window.getMemberList = window.lokiPublicChatAPI.getListOfMembers();
+
```
Comment (Bilb) [inline js/background.js]: to be removed, right?

File: js/views/inbox_view.js [hunk 2]
```diff
@@ -123,7 +123,8 @@
       });
 
       if (!window.storage.get('betaReleaseDisclaimerAccepted')) {
-        this.showBetaReleaseDisclaimer();
+        // Beta disclaimer disabled.
+        // this.showBetaReleaseDisclaimer();
```
Comment (Bilb) [inline js/views/inbox_view.js]: We are still in beta. why remove this?

File: ts/components/session/SessionScrollButton.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  count: number,
+}
+
+export class SessionScrollButton extends React.PureComponent<Props> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  public render() {
+    console.log(`My count is: ${this.props.count}`);
```
Comment (Bilb) [inline ts/components/session/SessionScrollButton.tsx]: console.log to be removed

File: ts/components/conversation/MemberList.tsx [hunk 4]
```diff
@@ -80,7 +78,8 @@ class MemberItem extends React.Component<MemberItemProps> {
         onClick={this.handleClick}
       >
         {this.renderAvatar()}
-        <span className="name-part">{nameAndPubKey}</span>
+        <span className="name-part">{name}</span>
+        <span className="pubkey-part">{pubkey}</span>
```
Comment (Bilb) [inline ts/components/conversation/MemberList.tsx]: If I remember correctly, I removed that one two weeks ago. Why do you need it there?

File: preload.js [hunk 5]
```diff
@@ -497,7 +504,7 @@ window.SMALL_GROUP_SIZE_LIMIT = 10;
 
 window.lokiFeatureFlags = {
   multiDeviceUnpairing: true,
-  privateGroupChats: false,
```
Comment (Bilb) [inline preload.js]: Are we ready to enable this feature? I mean is everything ready elsewhere on the app and backend and other platforms for privateGroupChats?

File: _locales/en/messages.json [hunk 6]
```diff
@@ -2645,20 +2661,19 @@
...
-    "message": "Add channel"
+    "message": "Join Open Group"
   },
   "enterChannelURL": {
-    "message": "Enter Channel URL"
+    "message": "Enter Open Group URL"
   },
   "channelUrlPlaceholder": {
-    "message": "https://yourchannel.lokinet.org"
+    "message": "https://myopengroup.lokinet.org"
```
Comment (Bilb) [inline _locales/en/messages.json]: Om mobile this is the chat.lokinet.org url which is displayed. Mayne we should make this the default too.

File: js/background.js (hunk 1 above)
Comment (vincentbavitz) [reply js/background.js]: No. For future implementation from Maxim.

File: js/views/inbox_view.js (hunk 2 above)
Comment (vincentbavitz) [reply js/views/inbox_view.js]: This is Simon's direction. He doesn't want the disclaimer anymore.

File: ts/components/conversation/MemberList.tsx (hunk 4 above)
Comment (vincentbavitz) [reply ts/components/conversation/MemberList.tsx]: Mentions needs to be split into two parts in order to render correctly as two span elements.

File: _locales/en/messages.json (hunk 6 above)
Comment (vincentbavitz) [reply _locales/en/messages.json]: Done

File: preload.js (hunk 5 above)
Comment (vincentbavitz) [reply preload.js]: Yes. This is Kees recommendation.

Comment (msgmaxim) [reply preload.js]: Why not just flip it to `true`?

File: preload.js [hunk 7]
```diff
@@ -59,6 +59,12 @@ window.isBeforeVersion = (toCheck, baseVersion) => {
   }
 };
 
+window.CONSTANTS = {
+  MAX_LOGIN_TRIES: 3,
+  MAX_PASSWORD_LENGTH: 32,
+  MAX_USERNAME_LENGTH: 20,
+};
```
[the remark this answers is not in the document]
Comment (vincentbavitz) [reply preload.js]: Now by accessing the window object, as window is defined in preload.

**Pull request merged**